### PR TITLE
add landing message

### PR DESCRIPTION
**Pull request by dasraev**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <title>welcome to Potionfy</title>
 </head>
 <body>
+<h1>Welcome to Potionfy . Enjoy your stay</h1>
```
Comment (dasraev-git) [inline index.html]: Could you remove 'Enjoy your stay' string and make 'Welcome to Potionfy' string red?

Comment (dasraev) [reply index.html]: sure , can do

Comment (dasraev-git) [reply index.html]: please,change the code as per my comments on the review

**Review by dasraev-git — CHANGES_REQUESTED** [review]

**Review by dasraev-git — APPROVED** [review]
Well done Harry!

**Pull request merged**